### ISSUE TITLE
xl: Healing a bucket should not populate an unformatted disk

### ISSUE THREAD

**Issue by vadmeste** [minio/minio#8188]
## Expected Behavior
Healing a bucket, with auto-healing, should not create a bucket in a disk although this latter is not formatted (in contrary to healing objects which does not do that)

## Current Behavior
Healing a bucket creates the bucket in the unformatted disk, the result is the disk becomes corrupted and not unformatted anymore

## Possible Solution
Do not know for sure

## Steps to Reproduce (for bugs)
1.

Apply the following code to accelerate auto healing after Minio startup

```
diff --git a/cmd/daily-sweeper.go b/cmd/daily-sweeper.go
index 99e65404a..39db67ee9 100644
--- a/cmd/daily-sweeper.go
+++ b/cmd/daily-sweeper.go
@@ -18,7 +18,6 @@ package cmd

 import (
        "context"
-       "math/rand"
        "sync"
        "time"

@@ -131,12 +130,12 @@ func dailySweeper() {
        }

        // Start with random sleep time, so as to avoid "synchronous checks" between servers
-       time.Sleep(time.Duration(rand.Float64() * float64(time.Hour)))
+       // time.Sleep(time.Duration(rand.Float64() * float64(time.Hour)))

        // Perform a sweep round each month
        for {
-               if time.Since(lastSweepTime) < 30*24*time.Hour {
-                       time.Sleep(time.Hour)
+               if time.Since(lastSweepTime) < 30*time.Second {
+                       time.Sleep(time.Second)
                        continue
                }

```
2. Create a bucket with mc and upload a file 
3. Remove on disk content 
4. Wait one minute until the next cycle of auto healing

## Context
Found after manual testing

## Regression
No

## Your Environment
All

**Comment by harshavardhana** [comment]
This should be fixed as well after #8408

**Comment by vadmeste** [comment]
> This should be fixed as well after #8408

Yes right, I already tested this use case when reviewing #8408, closing this since fixed.
